tests/tester.js

```diff
@@ -249,6 +249,7 @@ function runAll () {
   require('./tester.js')
   require('./genesishashes.js')
   require('./constantinopleSstoreTest.js')
+  require('./programCounter.js')
```

Comment:
Sorry, this is really completely our fault and I have directly created a test cleanup PR https://github.com/ethereumjs/ethereumjs-vm/pull/437 on this to avoid future confusion:

The `runAll()` method is not used for quite some time and is also not integrated into CI, so currently your tests are not run.

Can you just add a file to the `tests/api` folder instead? Then the tests are run along the `npm run testAPI` command.

Also: the naming with `programCounter.js` is far too specific. Can we change this to some to-be-expanded entry point to generally test `runCode` functionality and can you therefore change the filename to `runCode.js`?

Reply:
Done ;)